### PR TITLE
{2023.06}[foss/2023a] MetalWalls 21.06.1

### PR DESCRIPTION
**Pull request by Crivella**
Added `MetalWalls 21.06.1` with `foss/2023a` toolchain to `EasyBuild 4.9.2` easystack file

### CODE REVIEW

**Comment by ocaisa** [comment]
bot: build repo:eessi.io-2023.06-software arch:x86_64/generic

File: easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.0-2023a.yml [hunk 1]
```diff
@@ -74,3 +74,4 @@ easyconfigs:
       options:
         # see https://github.com/easybuilders/easybuild-easyconfigs/pull/20238
         from-pr: 20238
+  - MetalWalls-21.06.1-foss-2023a.eb
```
Comment (ocaisa) [inline easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.0-2023a.yml:77]: This is targetting the wrong EasyBuild release, you need EasyBuild 4.9.2

**Review by ocaisa — CHANGES_REQUESTED** [review]

File: easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.0-2023a.yml (hunk 1 above)
Comment (Crivella) [reply easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.0-2023a.yml]: Ups sorry, just realized i had bot files open and added it to the wrong one.
Fixed in [58fb56f](https://github.com/EESSI/software-layer/pull/642/commits/58fb56f2a8f2fb11d5df2b8332388ace82eda993)

**Comment by ocaisa** [comment]
bot: build repo:eessi.io-2023.06-software arch:x86_64/generic
bot: build repo:eessi.io-2023.06-software arch:aarch64/generic

**Review by ocaisa — APPROVED** [review]

**Comment by ocaisa** [comment]
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/haswell
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/skylake_avx512
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen2
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen3
bot: build repo:eessi.io-2023.06-software arch:aarch64/neoverse_n1
bot: build repo:eessi.io-2023.06-software arch:aarch64/neoverse_v1

**Review by ocaisa — APPROVED** [review]

**Comment by bedroge** [comment]
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen4

**Comment by bedroge** [comment]
I noticed that the `aarch64/generic` and `x64_64/generic` tarballs contain a `2023.06/init/eessi_environment_variables`. Looks like that's because these builds were done around the time that the tarballs of #623 were ingested, but the file probably wasn't available on the build hosts yet. A diff shows that the files included in these two tarballs are the same as the one that's already in our CVMFS repo, so we can safely ingest it (and overwrite the file with the same file).

**Review by bedroge — APPROVED** [review]

**Pull request merged**